### ISSUE TITLE
webdriver Manager , chrome version higher then 114

### ISSUE THREAD

**Issue by rajkrrathi** [angular/protractor#5563]
Since webdriver-manager is not supporting Chrome version > 114 and protractor has dev dependency on webdriver-manager.
How do we solve this issue?

Issue raised in webdriver-manager, regarding Chrome issue
https://github.com/angular/webdriver-manager/issues/523

**Comment by MattMoran12** [comment]
Yes I'm also curious how we can solve this issue. Downgrading an older chrome version is very difficult. This is blocking my protractor automated tests

**Comment by rajkrrathi** [comment]
For work around,  use the following link

https://github.com/angular/webdriver-manager/issues/517#issuecomment-1673232896

**Comment by jan-molak** [comment]
@rajkrrathi / @MattMoran12 - have a look at how I solved it in [Serenity/JS Protractor templates](https://github.com/serenity-js/?q=protractor-template&type=all&language=&sort=):
- instead of using webdriver-manager to download chromedriver, I use the [`chromedriver`](https://www.npmjs.com/package/chromedriver) module directly - see [package.json](https://github.com/serenity-js/serenity-js-mocha-protractor-template/blob/ebf58015e9e237611aab215c54702d38a99f7a58/package.json#L53)
- I then configure Protractor to use the chromedriver binary - see [protractor config](https://github.com/serenity-js/serenity-js-mocha-protractor-template/blob/ebf58015e9e237611aab215c54702d38a99f7a58/protractor.conf.js#L10)

You might also want to have a look at boilerplate [Serenity/JS + Protractor projects on GitHub](https://serenity-js.org/handbook/getting-started/project-templates/#protractor), which use this approach.

I hope this helps!
